can_tools/__init__.py

```diff
@@ -35,20 +35,10 @@ def all_subclasses(cls):
     )
 
 
-def non_abc_subclasses(cls):
-    out = list(x for x in all_subclasses(cls) if not inspect.isabstract(x))
-
-    # need to specifically remove some classes :(
```

Comment:
Instead of excluding these scrapers here, I've disabled them using `autodag = False` in the scrapers themselves. 

The Tableau class is a parent class so it shouldn't be included, but I don't have context as to why the PA scraper is disabled

Reply:
Hrm, yeah.  Not 100% sure. I remember there was a discrepancy with Philadelphia in the PA state dashboard, so maybe we just disabled the whole scraper?  Not sure.